src/Sensor/DiskMeasurement.java

```diff
@@ -0,0 +1,72 @@
+package Sensor;
+
+import org.hyperic.sigar.FileSystem;
+import org.hyperic.sigar.FileSystemUsage;
+import org.hyperic.sigar.Sigar;
+import org.hyperic.sigar.SigarException;
+
+/**
+ * Created by jak12 on 26.05.2017.
+ */
+public class DiskMeasurement extends Measurement {
+
+    FileSystem[] fileSystemList = null;
+
+    DiskMeasurement(Sigar _sigar) {
+        super(_sigar, "DiskUsage", "Megabytes");
+
+        prepareFileSystemInfo();
+        setMeasureMaxValue(calculateMaxValue());
+    }
+
+    void prepareFileSystemInfo(){
+        try {
+            fileSystemList = sigar.getFileSystemList();
+        } catch (SigarException sigarException) {
+            System.out.println(sigarException.getMessage());
+            sigarException.printStackTrace();
+        }
+    }
+
+    FileSystemUsage prepareFileSystemUsage(FileSystem fileSystem){
+        FileSystemUsage fileSystemUsage = new FileSystemUsage();
+
+        try {
+            fileSystemUsage.gather(sigar, fileSystem.getDirName());
+        } catch (SigarException sigarException) {
+            System.out.println(sigarException.getMessage());
+            sigarException.printStackTrace();
+        }
+        return fileSystemUsage;
+    }
+
+    String calculateMaxValue() {
```

Comment:
`calculateMaxValue` and `getActualMeasure` are pretty much this same, try to eliminate repeating code, look for
> DRY principle